.gitignore

```diff
@@ -14,3 +14,4 @@ secret
 .env.yaml
 .stage-env.yaml
```

Comment:
@NoahNsimbe i tried ignoring the folder `.idea/` but it seems to still be uploaded. Please advice

Reply:
Thanks  @HassanBahati. Try with an asterisk i.e `.idea/*` in order to exclude everything in the folder.

Reply:
@NoahNsimbe i have added an asterisk

Reply:
Looks like the files have not been ignored. We shall resolve this using a separate PR. Thanks @HassanBahati. Let me follow up on having this merged.